packages/request-node/src/request/confirmedTransactionStore.ts

```diff
@@ -0,0 +1,66 @@
+import { DataAccessTypes, LogTypes } from '@requestnetwork/types';
+import * as httpStatus from 'http-status-codes';
+
+import * as Keyv from 'keyv';
+
+/**
+ * Class for storing confirmed transaction information
+ * This class allow to store and serve the information when a transaction is confirmed
+ */
+export default class ConfirmedTransactionStore {
+  public store: Keyv<DataAccessTypes.IReturnPersistTransaction>;
+
+  /**
+   * Confirmed transaction store constructor
+   */
+  constructor() {
+    this.store = new Keyv<DataAccessTypes.IReturnPersistTransaction>();
```

Comment:
The store seems to be in-memory only. Is this what we want? (I'm not suggesting it's not, just want to make sure)

Reply:
As this is just a temporary storing, (after the client succeed to call this data they become useless), in-memory is enough. 
We will need a garbage collecter though, to not keep for ever useless data. (I jira It)

Reply:
what happens if the node is restarted ? What's the expected behavior of the `/getConfirmedTransaction` endpoint 
- for a non-confirmed tx
- for a confirmed tx

Reply:
discussed. Ok for now. Jira for a long term solution.